src/ui/modals/components/EditNote.svelte

```diff
@@ -22,16 +22,33 @@
 </script>
 
 <ModalLayout title={$i18n.t("modals.note.edit.title")}>
+  {#if !editableFields.length}
+    <Callout
+      title={$i18n.t("modals.note.edit.no-editable-fields.title")}
+      icon="info"
+      variant="info"
+    >
```

Comment:
Extracting the condition and `Callout` from `ModalContent`, to avoid an extra separator (line).